example/cypress/e2e/data-table.cy.ts

```diff
@@ -196,35 +196,38 @@ describe('DataTable', () => {
     });
   });
 
-  it('checks for data tables with sticky header', () => {
-    cy.get('div[data-cy=table-expandable-0]').as('sticky');
+  it.only('checks for data tables with sticky header', () => {
```

Comment:
I guess only was so that you could check the test? for CI it should be removed though.

Reply:
Was for me to easily test on local... will remove it